controllers/storagecluster/provider_server.go

```diff
@@ -145,6 +145,12 @@ func (o *ocsProviderServer) createDeployment(r *StorageClusterReconciler, instan
 
 func (o *ocsProviderServer) createService(r *StorageClusterReconciler, instance *ocsv1.StorageCluster) (reconcile.Result, error) {
 
+	if instance.Spec.ProviderAPIServerServiceType == corev1.ServiceTypeClusterIP || instance.Spec.ProviderAPIServerServiceType == corev1.ServiceTypeExternalName {
+		err := fmt.Errorf("service of Type %s is not supported by the operator", instance.Spec.ProviderAPIServerServiceType)
```

Comment:
Can you pls mention that only 2 types of services are supported for now which are nodeport and loadbalancer.